src/splinterdb.c

```diff
@@ -227,38 +229,95 @@ splinterdb_init_config(const splinterdb_config *kvs_cfg, // IN
  * Internal function for create or open
  */
 int
-splinterdb_create_or_open(const splinterdb_config *kvs_cfg,      // IN
-                          splinterdb             **kvs_out,      // OUT
-                          bool                     open_existing // IN
+splinterdb_create_or_open(splinterdb_config *kvs_cfg,      // IN
```

Comment:
The reason why `kvs_cfg` is losing the `const` attribute is due to the change(s) below on L310 & 311. See reasons below.

Reply:
Resolved. Reworked interfaces to re-introduce const splinterdb_config *kvs_cfg.